tests/test_wave_01.py

```diff
@@ -30,9 +30,9 @@ def test_create_no_title_movie():
     new_movie = create_movie(movie_title, genre, rating)
 
     # Assert
-    assert new_movie is None
+    assert new_movie == None
```

Comment:
This check works and passes, but the preferred comparison operator is `is` for checking whether a variable is None. [This article](https://www.pythontutorial.net/advanced-python/python-none/) gives a good explanation.